core/src/main/java/org/mskcc/cbio/portal/util/ExtendedMutationUtil.java

```diff
@@ -175,7 +179,7 @@ public static boolean isValidProteinChange(String proteinChange)
 		boolean invalid = proteinChange == null ||
 		                  proteinChange.length() == 0 ||
 		                  proteinChange.equalsIgnoreCase("NULL") ||
-		                  proteinChange.equalsIgnoreCase(NOT_AVAILABLE);
+		                  NOT_AVAILABLE.contains(proteinChange);
```

Comment:
now this is case sensitive? It wasn't. Please fix, e.g. change list to all lower case and do:
`NOT_AVAILABLE.contains(proteinChange.toLowerCase())`